docs: Remove TOC duplication on future work section pages (#1038)
thousandbrainsproject/tbp.monty: docs/future-work/monty-demos.md

title: Monty Demos
description: These are efforts to develop real-world demos of Monty's capabilities.
---
In this section we list items related to demoing Monty's capabilities on real-world applications. While we are mostly focused on developing a general purpose algorithm that isn't optimized for any particular application, we like to get insights into how well Monty actually does outside our test environments. 

> Please feel free to share on our [Forum](https://forum.thousandbrains.org/) or [Project Showcase](../community/project-showcase.md) page if you have tested Monty in a new setting. Especially if you struggled with this or ran into limitations, we are very interested in those.

Reviewer: ```suggestion
Please feel free to share on our [Forum](https://forum.thousandbrains.org/) or [Project Showcase](../community/project-showcase.md) page if you have tested Monty in a new setting. We are especially interested in hearing about any challenges or limitations you encounter.
```
